draft-shahzad-scim-device-model.mkd

```diff
@@ -605,113 +699,100 @@ representation is provided in Section [](#zigbee-extension-schema-openapi-repres
 
 
 
-## The Endpoint Applications Extension Schema
+## The Endpoint Applications Extension Schema {#endpointsappext-schema}
 
 Sometimes non-IP devices such as those using BLE or Zigbee require an
 application gateway interface to manage them. SCIM clients MUST NOT
 specify this to describe native IP-based devices.
 
-The device schema is a base schema upon which specific onboarding // FIXME no it's not
-technology schemas are built. This extension provides two complex
-attributes deviceControl, and dataReceiver to represent
-two types of partner endpoint applications, device
-control, and data receiver applications, respectively. 
-The extension is identified using the following schema URI:
-
-urn:ietf:params:scim:schemas:extension:endpointApps:2.0:Device // FIXME
+endpointAppsExt provides the list application that connect to 
```

Comment:
For this `endpointAppsExt` _extension_ to be present for each device, we need to also be able to set a device or device group in the `EndpointApp` _resource_ to allow for access control. 
Without device groups in the `EndpointApp`, we have no way of knowing which endpoint apps are applicable for a particular device.